cmd/notary/tuf.go

```diff
@@ -440,17 +448,36 @@ func (t *tufCommander) tufStatus(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if t.resetStatus {
+		return cl.Clear(t.archiveChangelist)
```

Comment:
Might be nice to have a `--reset --interactive` but I don't think it's necessary for this PR. We have other things to get done right now :-)